charts/kuberpult/templates/cd-service.yaml

```diff
@@ -80,6 +80,8 @@ spec:
           value: {{ required ".Values.git.url is required" .Values.git.url | quote }}
         - name: KUBERPULT_GIT_BRANCH
           value: {{ .Values.git.branch | quote }}
+        - name: KUBERPULT_ARGOCD_BASE_URL
```

Comment:
I would let the frontend-service handle this request as it is strictly for the frontend.